frontend/src/types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+export {};
+
+declare global {
+  // eslint-disable-next-line no-unused-vars
+  interface Window {
+    YonseiSpaceSystem: any;
+  }
+}
```

Comment:
여기를 `any` 로 두는 것보다 API를 명시적으로 표시해 주는게 더 좋지 않을까?

```suggestion
declare global {
  // eslint-disable-next-line no-unused-vars
  interface Window {
    YonseiSpaceSystem: {
        login: (id:string, pw:string) => Promise<boolean>;
    };
  }
}
```